app/models/feedback_reporters/support_reporter.rb

```diff
@@ -16,7 +16,10 @@ def custom_zoho_fields
     {
       "cf_archive_version" => site_revision.presence || "Unknown site revision",
       "cf_rollout" => rollout.presence || "Unknown",
-      "cf_user_agent" => user_agent.presence || "Unknown user agent"
+      "cf_user_agent" => user_agent.presence || "Unknown user agent",
+      "cf_ip" => ip_address.presence || "Unknown",
```

Comment:
Would it make sense to do "Unknown IP" and "Unknown URL" to match the abuse reporter? Although "URL" could be "Referer," too.

...actually, I wonder if we should check with Support/PAC and see if they should both use `cf_url` as the field name? I know they sometimes transfer tickets between their Zoho departments, and I don't know if a) the field names need to match for information to transfer, or b) if they'd want the URL/referer to transfer. 🤔

Reply:
I'm checking with Support and PAC chairs about the field names, btw.

Reply:
(I'm happy to change it back, but since part of the ticket is about handing tickets to PAC I think it makes sense to use the same name to make that as easy as possible.)

Reply:
We should change it back! Support has had a cf_url field for a while now, for the sole purpose to storing URLs from transferred PAC tickets, and it has been working fine.